tracing.go

```diff
@@ -70,6 +77,16 @@ func TracingMiddleware(h message.HandlerFunc) message.HandlerFunc {
 			span.End()
 		}()
 
+		eIDString := msg.Metadata.Get(spanEventIDKey)
+		eID, _ := strconv.ParseInt(eIDString, 10, 64)
+
+		messageBytes := []byte(msg.Payload)
```

Comment:
As far as I remember it's alrady []byte :P

Reply:
https://github.com/czeslavo/watermill-opencensus/pull/6/commits/0c281d9a162452456cbca755e765cb33add72009